src/Controller/HearingController.php

```diff
@@ -195,7 +251,28 @@ public function hearingPostForward(CaseEntity $case, HearingPost $hearingPost):
             throw new HearingException();
         }
 
-        // TODO: Send digital post envelope containing hearing post
+        // Create DigitalPost
+        $digitalPost = new DigitalPost();
+        $digitalPost->setDocument($hearingPost->getDocument());
+        $digitalPost->setEntityType(get_class($case));
+        $digitalPost->setEntityId($case->getId());
+
+        // Handle attachments
+        $attachments = $hearingPost->getAttachments();
+
+        foreach ($attachments as $attachment) {
+            $digitalPostAttachment = new DigitalPostAttachment();
+            $digitalPostAttachment->setPosition($attachment->getPosition());
```

Comment:
This is handled by `DigitalPost` (cf. https://github.com/itk-dev/naevnssekretariatet/blob/develop/src/Entity/DigitalPost.php#L230-L238)

Reply:
Fixed in a3d58663c826d21bd47a98dee75f3b1ba9610973.